src/page-multi-facility/HistoricalCasesChart/BarChartTooltip.tsx

```diff
@@ -47,19 +50,22 @@ const TooltipSubtitle = styled.div`
 
 const BarChartTooltip: React.FC<TooltipProps> = ({ summary }) => {
   const summaryData = summary[0];
-  const { data, value } = summaryData;
+  const { data } = summaryData;
   return (
     <TooltipContainer>
       {data.missing ? (
         <TooltipTitle>No data</TooltipTitle>
       ) : (
-        <TooltipTitle>{value} cases</TooltipTitle>
+        <>
+          <TooltipTitle>{data.cases} cases</TooltipTitle>
+          <TooltipTitle>{data.population + data.cases} residents</TooltipTitle>
+        </>
       )}
 
       <TooltipDate>
         <DateMMMMdyyyy date={data.observedAt} />
       </TooltipDate>
-      <TooltipSubtitle>Click to update</TooltipSubtitle>
+      <TooltipSubtitle>Change</TooltipSubtitle>
```

Comment:
I think we might want to keep this as `Click to update` unless something has changed recently...this is different than the mock and comes from a slack chat I had with Serena after we talked about how it might be difficult to click on link within the hovered tooltip, so the idea was to change it to make it clear the user should click on the bar instead to open the modal.

Reply:
Ok, the designs in #405 say "Change" so I updated it. Will leave it as "Click to update".